src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-CSharp.csproj.in

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="${MicrosoftAspNetCoreAppPackageVersion}" />
```

Comment:
Revert this change. Plan of record for now is to keep implicit package versions.

Reply:
This change is the primary fix in this PR. It fixes [this failure](https://github.com/aspnet/Templating/issues/640). If we're not doing this then how do you recommend we fix that?

Reply:
The versionless package reference is a product feature. What is *supposed* to happen (but may not be working) is that we generate this file to override the version used as the implicit version on the package ref.

https://github.com/aspnet/Templating/blob/0435add47ad7a10d985edd1f48a8851e6041cedf/test/TemplateTests.props.in#L9-L10

See also https://github.com/aspnet/websdk/blob/release/2.2.1xx/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets#L64

Reply:
Lets talk about this offline, I think I'm missing/not grasping something.

Reply:
How can we both have the items in this file and delete them, as you recommended above?